plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/app/PluginStrings.js

```diff
@@ -232,6 +232,9 @@ Ext.define('NX.coreui.app.PluginStrings', {
     ADMIN_BLOBSTORES_LIST_NEW_BUTTON: 'Create blobstore',
     ADMIN_BLOBSTORES_LIST_NAME_COLUMN: 'Name',
     ADMIN_BLOBSTORES_LIST_TYPE_COLUMN: 'Type',
+    ADMIN_BLOBSTORES_LIST_BLOB_COUNT_COLUMN: 'Blob Count',
```

Comment:
Component isn't right, since in NX 3 a component (e.g. 'junit 2.1') could have several blobs (at least one for each of its assets, such as its POM, .jar, etc). Other kinds of content, too, could get stuffed in the blob store (e.g. cached version of locally-generated content).